### ISSUE TITLE
Standardize date formatting across the app

### ISSUE THREAD

**Issue by emanuelegurini** [Balastrong/myntenance#28]
I want all dates within the app to be formatted consistently.

**Comment by Balastrong** [comment]
Yep, we definitely need an util function somewhere to format all dates. This would also fix the issue of using toLocaleDateString combined with server side rendering, not ideal.

If anyone wants to pick this issue just let me know :)
